### ISSUE TITLE
Android still supported?

### ISSUE THREAD

**Issue by mrbluecoat** [NeoRaider/fastd#17]
https://github.com/rlei/fastd-android says it was merged in but I don't see https://git.universe-factory.net/fastd/tree/doc/build-fastd-android.sh

Is this repo a fork?

**Comment by NeoRaider** [comment]
Hi, Android support still exists in the code itself, but the old build script relied on the CMake build system. With release v20, fastd switched from CMake to Meson. I don't know enough about Android to implement a new build script, but I'll gladly accept contributions for it.

**Comment by marek22k** [comment]
Maybe @rlei knows something. He/She appears in the commit history of the Android project.

**Comment by rlei** [comment]
Yes I contributed the android support but it's quite some years ago 😅 it'll probably need a new maintainer as I've stepped away from Android dev for a long time.

**Comment by kameneken** [comment]
Hey guys, I am working on an Android app for fastd which will soon be releasing it on Google Play for closed testing. If you're interested, you can send your Google Play account to kameneken@gmail.com so that I can send you an invitation to the beta testing.
![Screenshot_20240401-105058](https://github.com/neocturne/fastd/assets/166382900/d15f6218-63a1-460a-9214-e1a5c955e11d)
![Screenshot_20240401-105144](https://github.com/neocturne/fastd/assets/166382900/bc4c3868-d9c6-494e-92c9-e297db90f80a)
![Screenshot_20240401-105249](https://github.com/neocturne/fastd/assets/166382900/c9231ee3-850a-4502-80ec-59ce685422d0)
